taker-frontend/src/components/MainPageLayout.tsx

```diff
@@ -61,7 +61,8 @@ export function MainPageLayout(
                                 textAlign="center"
                                 bg={useColorModeValue(BG_LIGHT, BG_DARK)}
                                 maxWidth={VIEWPORT_WIDTH_PX}
-                                marginTop={`${HEADER_HEIGHT}px`}
+                                marginTop={(HEADER_HEIGHT
+                                    + (outdatedWarningIsVisible || incompatibleWarningIsVisible ? 50 : 0)) + "px"}
```

Comment:
we can avoid the magic number, otherwise if we change the header height const this would get out of sync 

```suggestion
                                    + (outdatedWarningIsVisible || incompatibleWarningIsVisible ? HEADER_HEIGHT : 0)) + "px"}
```

Reply:
well, its actually not the HEADER_HEIGHT, but incidentally the height is the same.. its the space that the disclaimer takes and we do not have a constant for that (at least I haven't found one).

Reply:
https://github.com/itchysats/itchysats/blob/real-world-ethusd-values/taker-frontend/src/components/NavBar.tsx#L355 - it seems it is the same one?

Reply:
You are right, still its not the same height due to the padding 4.. but anyhow.. I'll change it.